finder-app/Makefile

```diff
@@ -0,0 +1,8 @@
+CC=gcc
+CFLAGS=-Wall -Werror -g -O0
+
+writer:
+	$(CC) $(CFLAGS) writer.c -o writer
```

Comment:
@ECleverito see notes in lecture video/slides about how to support cross compile here.